packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -1180,29 +1181,21 @@ namespace Ifpack2 {
       local_ordinal_type pack_nrows_sub = 0;
       if (jacobi) {
         IFPACK2_BLOCKHELPER_TIMER("compute part indices (Jacobi)", Jacobi);
+        // Jacobi (all lines have length 1) means that A_n_lclrows == nparts,
```

Comment:
The changes here are supposed to keep all the resulting views exactly the same as before. It's just simplifications based on the fact that ``ipnrows == 1`` in the Jacobi case. It was also helpful to convince myself that the ``lclrow`` mapping is the identity in this case, since it simplifies code in the numeric and solve.